Gemfile.lock

```diff
@@ -576,10 +576,10 @@ GEM
     ruby-next-core (1.0.3)
     ruby-progressbar (1.13.0)
     ruby2_keywords (0.0.5)
-    runger_actions (0.23.0)
-      memo_wise (>= 1.7, < 2)
-      rails (>= 6, < 8)
```

Comment:
I think that this will make it possible to upgrade to Rails 8.0 (or at least will get us closer).